GDJS/Runtime/CustomRuntimeObject.ts

```diff
@@ -91,13 +75,26 @@ namespace gdjs {
       );
       this._renderer = this._createRender();
 
-      this._createDefaultSizeIfNeeded(objectData);
-      this._instanceContainer.loadFrom(objectData);
+      this._initializeFromObjectData(objectData);
 
       // The generated code calls onCreated at the constructor end
       // and onCreated calls its super implementation at its end.
     }
 
+    private _initializeFromObjectData(
+      objectData: ObjectData & CustomObjectConfiguration
+    ) {
+      const eventsBasedObjectData = this._runtimeScene
+        .getGame()
+        .getEventsBasedObjectData(objectData.type);
+      if (!eventsBasedObjectData) {
+        logger.error('loadFrom was called without an events-based object');
```

Comment:
Careful about the error message. Custom objects are complex so let's be super clear and specific:
```suggestion
        logger.error(`A CustomRuntimeObject was initialized (or re-initialized) from object data referring to an non existing events based object data with type "${objectData.type}".`);
```